rds_database_cluster/rds_database_cluster.tf

```diff
@@ -14,6 +14,7 @@ resource "aws_ssm_parameter" "db_root_master_username" {
   name  = "${lookup(var.optional_parameters, "parameter_store_path", "${var.custom_identifier}")}/${lookup(var.optional_parameters, "parameter_store_username_key", "Username")}"
   type  = "String"
   value = "${lookup(var.optional_parameters, "master_username", "root")}"
+  tags  = "${var.optional_ssm_parameter_tags}"
```

Comment:
wont rds_database_instance need it too ?

Reply:
not right now